src/feature.h

```diff
@@ -26,6 +26,8 @@
 
 /* Provided feature names */
 #define FEATURE_OUTPUT_FILESTORE    "output::file-store"
+#define FEATURE_JA3                 "ja3"
+#define FEATURE_JA4                 "ja4"
```

Comment:
ja4 line was added in the wrong commit :-p

Reply:
Oops!

Reply:
Moved to correct commit.